src/SFA.DAS.ApprenticeFeedback.Domain/SFA.DAS.ApprenticeFeedback.Domain.csproj

```diff
@@ -4,6 +4,10 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Interfaces\IExitSurveyAttributeContext.cs" />
```

Comment:
Is this correct? removing it from Compile?

Reply:
The file was removed from the project, it should have been deleted. Sorted now.